crypter/shield-crypt/shield-crypt.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"crypto/cipher"
+	"encoding/hex"
+	"encoding/json"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/jhunt/go-cli"
+	"github.com/starkandwayne/shield/crypter"
+)
+
+type CLIOpts struct {
+	EncryptShort bool `cli:"-e"`
+	DecryptShort bool `cli:"-d"`
+	EncryptFull  bool `cli:"--encrypt"`
+	DecryptFull  bool `cli:"--decrypt"`
```

Comment:
`go-cli` lets you do this:

```
struct {
  Encrypt bool `cli:"-e, --encrypt"`
  Decrypt bool `cli:"-d, --decrypt"`
}
```

We should probably make use of that to simplify handling in main.